backend/server/src/models/tournamentModel.ts

```diff
@@ -39,12 +68,14 @@ const tournamentSchema = new Schema<Tournament & Document>(
     },
     maxPlayers: { type: Number, required: true },
     players: [{ type: String }],
-    matchSchedule: [{ type: Schema.Types.ObjectId, ref: "Match" }], // Reference to Match documents
+    matchSchedule: [{ type: String }], // Reference to Match documents
```

Comment:
Why is this a string type?

Reply:
Modified.

`matchSchedule: [{ type: Types.ObjectId }],`